backend/Gemfile

```diff
@@ -41,7 +41,7 @@ end
 
 group :test do
   gem 'shoulda-matchers'
-  gem 'simplecov', require: false
+  gem 'simplecov', '0.17', require: false
```

Comment:
Algum motivo para ser exatamente essa versão?

Reply:
@leonardobrito , this one is the last one compatible with codeclimate